k9mail/src/main/java/com/fsck/k9/activity/compose/ComposeCryptoStatus.java

```diff
@@ -102,6 +103,9 @@ public boolean isSigningEnabled() {
         return cryptoMode != CryptoMode.DISABLE && signingKeyId != null;
     }
 
+    public boolean isCompatModeEnabled() {
+        return enableCompat;
+    }
 
```

Comment:
I have not considered s/mime here. do you have any other way to handle this in mind?

about the other option, I don't think that one needs to be exposed to the user. I'm not even sure we need to support it...